webapp/client/src/components/SecondaryAnchorButton.js

```diff
@@ -3,7 +3,7 @@ import styled from "styled-components";
 import addPlausibleGoal from "../lib/helpers";
 
 const AnchorSecondary = styled.a`
-  padding: 1rem 1.25rem calc(1rem - 4px) 1.25rem; /* The calculation subtracts the border-bottom height. We need a border-bottom for the focus state. */
+  padding: 18px 24px 18px 24px;
```

Comment:
I guess this was copied from earlier? I think that the padding leads to a problem when it is in focus state (it toggles).

Reply:
Your right...